v2/svid/common/optional/optional.go

```diff
@@ -0,0 +1,16 @@
+package optional
+
+// SVIDOptionals contains optional fields for SVIDs.
+type SVIDOptionals struct {
```

Comment:
I don't think it is likely that we'll gain any more "optional" fields on the SVIDs. I think I'd probably be more comfortable with just including the Hint field in each of the SVID structs.

Reply:
You are right, I did a bit of over eng here